roles/vector/tasks/repo.yml

```diff
@@ -1,25 +1,60 @@
 ---
 
-- block:
-    - name: Add vector gpg key to apt
-      become: true
-      apt_key:
-        url: "{{ vector_repo_key }}"
+- name: Fetch OS dependent variables
```

Comment:
I see no OS specific vars, is thin needed somewhere?

Reply:
Your right. I removed this task.